playwright/pages/AddPatientAttachmentPage.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable radix */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable prettier/prettier */
+const BasePage = require('../utilities/UtilityMethods');
+
+class AddPatientAttachmentPage extends BasePage {
+    constructor(page) {
+        super(page);
+        // Define selectors or other properties specific to the labs page if needed
+    }
+
+async AddPatientAttachment()
+{ 
+     
+    await this.page.goto('http://openihp-qa-web.southafricanorth.cloudapp.azure.com:8390/session');
```

Comment:
URL should not be hardcoded. This cannot possibly have worked on local.

Reply:
this page wrote initially now i have never used it i have removed the whole page